src/app/footer/footer.component.css

```diff
@@ -17,18 +17,70 @@
 .container-fluid,
 a {
   background-color: #21335b;
-  color: white;
+  color: #e8eaf6;
 }
 
 .container-fluid {
   padding-top: 20px;
 }
 
-.container-fluid,
 ul {
   list-style: none;
 }
 
-.footer-list li:first-child {
-  font-weight: bold;
+li {
+  line-height: 26px;
+}
+
+/* the title of each column */
+.title {
+  /* used for making the text semi-bold */
+  text-shadow: 0 0 0 #c5cae9;
+  color: #c5cae9;
+  text-transform: uppercase;
+  padding: 10px 0;
```

Comment:
I will beat Gary to his comment and say we should avoid the use of pixels in css.

Reply:
@coverbeck I see, as I noticed many other stylings are in `px` as well. Are you suggesting we covert them to `rem`?

Reply:
> @coverbeck I see, as I noticed many other stylings are in `px` as well. Are you suggesting we covert them to `rem`?

Yes :) .  Rems work better with different screen sizes/resolutions.

See https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide , about the 10th item.

And yes, we have a ton of styles using px, but we're trying to avoid adding additional ones going forward.

Reply:
Ah, I should've probably mentioned this months ago. I think Emily had a PR that fixes px for different screen sizes. though it wouldn't really hurt to just keep using rem.